web/demos/button/button-active.dart

```diff
@@ -0,0 +1,12 @@
+part of over_react.web.demos;
```

Comment:
Can these be there own libraries? It is the preferred way of organizing packages: https://www.dartlang.org/guides/libraries/create-library-packages#organizing-a-library-package.